package-lock.json

```diff
@@ -1,52 +1,8 @@
 {
   "name": "mailersend",
   "version": "1.1.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Looks like you are using NPM v6. You should use NPM v7, which is the current version and this package-lock file should be updated based on the current version. The changes done in this package-lock file might not work in V7. Please refer to this doc: https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json